pkg/helm/request.go

```diff
@@ -0,0 +1,8 @@
+package helm
+
+type HelmRequest struct {
```

Comment:
please add `json` tags for unmarshalling.

Reply:
added